app/src/androidTest/java/org/projectbuendia/client/ui/chart/PatientChartActivityTest.java

```diff
@@ -271,8 +271,8 @@ public void testEncounter_latestEncounterIsAlwaysShown() {
             // TODO: implement IdlingResource for webview to remove this sleep.
             // Wait a bit for the chart to update it's values.
             try{
-                Thread.sleep(30000);
-            } catch (InterruptedException e){}
+                Thread.sleep(5000);
+            } catch (InterruptedException ignored){}
```

Comment:
That's got a specific meaning - the linter in Android Studio flags empty catch blocks, unless the exception is deliberately named "ignored".